### PR TITLE
[REVIEW] Add init files for cython pkgs and fix setup.py

### PR DESCRIPTION
**Pull request by aucahuasi**

### CODE REVIEW

File: .pydevproject [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (kkraus14) [inline .pydevproject]: Remove this file

File: python/cudf/setup.py [hunk 2]
```diff
@@ -93,6 +93,11 @@
         "cudf._lib.arrow": ["*.pxd"],
         "cudf._libxx": ["*.pxd"],
         "cudf._libxx.cpp": ["*.pxd"],
+        "cudf._libxx.cpp.column": ["*.pxd"],
+        "cudf._libxx.cpp.io": ["*.pxd"],
+        "cudf._libxx.cpp.scalar": ["*.pxd"],
+        "cudf._libxx.cpp.table": ["*.pxd"],
+        "cudf._libxx.cpp.wrappers": ["*.pxd"],
```
Comment (kkraus14) [inline python/cudf/setup.py]: Could you check if a wildcard works here instead of specifying each folder? I.E. could we do `cudf._libxx.cpp.*`?

Comment (aucahuasi) [reply python/cudf/setup.py]: It works using a wildcard, done ;)

File: .pydevproject (hunk 1 above)
Comment (aucahuasi) [reply .pydevproject]: ups ... removed

**Comment by kkraus14** [comment]
@aucahuasi add a changelog entry and this is good to go 😄

**Pull request merged**